src/software/firmware/srcs/screen.cpp

```diff
@@ -78,8 +78,14 @@ void displayCurrentInformation(uint16_t peakPressure, uint16_t plateauPressure,
     screen.setCursor(0, 3);
     char message[SCREEN_LINE_LENGTH + 1];
 
-    (void)snprintf(message, SCREEN_LINE_LENGTH + 1, "%2u    %2u    %2u  meas", peakPressure / 10u,
+    // Case plateau is not detected
+    if (plateauPressure == UINT16_MAX){
+        (void)snprintf(message, SCREEN_LINE_LENGTH + 1, "%2u    ?    %2u  meas", peakPressure / 10u,
```

Comment:
I think there is a missing `space` or  `?` in this message. 
Line 86, pattern ` "%2u    %2u    %2u  meas"` has a length of 20 characters. (2 + 4 spaces + 2 + 4 spaces + 2 + 2 spaces + 4 letters).
Line 83 we have only 2 + 4 spaces + 1 + 4 spaces + 2 + 2 spaces + 4 letters = 19 characters.
I suggest to add an empty space before `?`.

Reply:
Done! I caught it too ^^